migrations_lockfile.txt

```diff
@@ -10,3 +10,4 @@ nodestore: 0002_nodestore_no_dictfield
 replays: 0003_add_size_to_recording_segment
 sentry: 0565_fix_diff_env_dupe_alerts
 social_auth: 0002_default_auto_field
+hybridcloud: 0001_add_api_key_replica
```

Comment:
Do we have a rule of thumb for what models should/should not live here?

Is it just model replicas?
- If so, at the risk of bikeshedding, maybe something on along the lines of `hybridcloud_replica` or further split into `region_replica`/`control_replica`?
- If not, what else would live here?

Reply:
My rule of thumb is Conway's law.
We are hybrid cloud, we have shared concerns, and we can maintain whatever exists it as it moves forward.
It is, most likely we need to coordinate internally as well.

Reply:
My personal preference would be to keep this feature/product centric like the others rather than team-centric, even if it pretty much ends up holding the same models in the end. Basically "hybrid cloud the feature" over "hybrid cloud the team" so that it's clear to other folks adding new replicas that this is where those models should probably go. That said, none of this affects the code as written - just how it affects other folks longer term. LGTM